index.html

```diff
@@ -53,35 +53,40 @@ <h4 class="icon">LET’S CONNECT</h4>
         </li>
       </ul>
     </section>
-  <section>
-    <div class="card-contain">
-      <ul class="cards-list flex_column">
-        <li class="card-item flex-culomn">     
-<img class="card-image" src="img/Snapshoot-Portfolio1.png" alt="Portfolio Portfolio">
-<div class="list-container">
-  <h2>Tonic</h2>
-  <div class="lists">
-    <span class="lists-item obect1">Canopy</span>
-    <span class="lists-item">back end Dev</span>
-    <span class="lists-item">2015</span>
-  </div>
-</div>
-<div class="text-holder">
-  <p>
-    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-  </p>
-  </div>
-  <div class="language-list-holder">
-    <ul class="language-container">
-      <li class="language-item"><span>JavaScript</span></li>
-      <li class="language-item"><span>Html</span></li>
-      <li class="language-item"><span>Css</span></li>
-    </ul>
-  </div>
-  <button type="button" class="click-btn">See project</button>
-</li>
-</ul>
-</div>
-  </section>
+    <section>
+      <div class="card-contain">
+        <ul class="cards-list flex_column">
+          <li class="hold-item flex-culomn">
+            <img
+              class="hold-image"
+              src="img/Snapshoot-Portfolio1.png"
+              alt="Portfolio Portfolio"
+            />
+            <div class="list-container">
+              <h2>Tonic</h2>
```

Comment:
- [ ] Great job so far 👍 Kindly make sure that the background image of each card is white throughout so as to match the Figma template. The sides of your cards do not have a white background.
- Your design:
![Annotation 2022-03-26 195319](https://user-images.githubusercontent.com/85113567/160253573-07209f7f-acc9-4511-883f-46af0465498f.jpg)
- Figma design:
![Annotation 2022-03-26 195302](https://user-images.githubusercontent.com/85113567/160253584-f0a53438-636a-4603-ae9b-b64a2ac47943.jpg)

Reply:
Hello, @Kingstalux I have made the requested changes. Thank you for reviewing my PR. 🤝